tests/Service/ProgrammesAggregationService/FindProgrammesChildrenByTypeTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Service\ProgrammesAggregationService;
+
+use BBC\ProgrammesPagesService\Domain\Entity\Programme;
+use InvalidArgumentException;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Service\ProgrammesAggregationService::findProgrammesChildrenByType
+ * @group service
+ * @group programmesAggregation
+ */
+class FindProgrammesChildrenByTypeTest extends AbstractProgrammesAggregationTest
+{
+    /**
+     * @expectedException InvalidArgumentException
+     */
+    public function testCannotGetBrandTypesChildrenForAnyAncestry()
```

Comment:
I tend to put the error tests towards the bottom - making sure stuff works is the important thing to look for as you're scrolling through